### ISSUE TITLE
No option to enable the DirectInput/Xinput function in Kii on the configurator

### ISSUE THREAD

**Issue by BOTButtcheeks** [kiibohd/KiiConf#37]
When the Ergodox infinity is plugged in it shows that it is a game controller (possibly DirectInput/Xinput) as well as the obvious keyboard and Com port.

Could something be implemented into the configurator to access this functionality and pass the keystrokes as controller input?

**Comment by haata** [comment]
Yay, I'm starting to get requests for this :D 

What sort of controller should I try to spoof the keyboard as?
For some thing I may have to do something custom, but in general, would say an XBox 360 controller work? Or is the standard something else now?

**Comment by BOTButtcheeks** [comment]
currently Xinput (Xbox360) is the current standard that is most compatible with almost everything now, however Xbox controllers are expected to have a certain number of axes and buttons and that cant be exceeded.

if only one of the two is doable It might be worth using DirectInput and leaving emulating the Xbox360 controller to programs like X360ce or even the easier JoytoKey (especially since keyboards don't have analog axes anyway)

I personally will be using joytokey to make temporary maps without having to flash every time I need to do something, so having access to all the buttons would be quite necessary, but that is just my usage.

tl;dr: Use Directinput, the standard that cheap Chinese or older controllers go off as it allows for 32 buttons as opposed to 10
